ci/jenkins/test.sh

```diff
@@ -232,7 +232,7 @@ function deliver_antrea_windows {
     chmod -R g-w build/images/ovs
     chmod -R g-w build/images/base
     DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-ubuntu-all.sh --pull
-    if [[ "$TESTCASE" =~ "networkpolicy" ]]; then
+    if [[ "$TESTCASE" =~ "networkpolicy-no" ]]; then
```

Comment:
What is networkpolicy-no for?

Reply:
This is used to let Antrea Windows agent start as Daemonset for Windows NetworkPolicy test.